tests/integration/kubernetes/k8s-trusted-ephemeral-data-storage.bats

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bats
+# Copyright (c) 2025 Microsoft Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+load "${BATS_TEST_DIRNAME}/lib.sh"
+load "${BATS_TEST_DIRNAME}/../../common.bash"
+load "${BATS_TEST_DIRNAME}/confidential_common.sh"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
+
+setup() {
+    is_confidential_runtime_class || skip "Test only supported for CoCo"
```

Comment:
To skip on s390x https://github.com/kata-containers/kata-containers/pull/10559#issuecomment-2594671838